litex/soc/integration/soc_core.py

```diff
@@ -190,6 +192,11 @@ def __init__(self, platform, clk_freq,
             if timer_uptime:
                 self.timer0.add_uptime()
 
+        # Add CFU
+        if cfu_filename:
+            assert(cpu_type == "vexriscv")
+            self.cpu.add_cfu(cfu_filename=cfu_filename)
```

Comment:
Also assert `("cfu" in cpu_variant)`.   The assertion will be less confusing than getting "`named port connection 'CfuPlugin_bus_cmd_ready' does not exist`" from Vivado.

Eventually when I've build a +cfu version of every VexRiscv variant, the code can modify the variant here if needed (infer the "+cfu").